### PR TITLE
Simplify contribute.md

### PR DESCRIPTION
**Pull request by Riksu9000**
Greatly simplified the contributing doc to make it a lighter read.

### CODE REVIEW

File: doc/contribute.md [hunk 1]
```diff
@@ -1,61 +1,19 @@
...
- - Test your modifications on the actual hardware
- - Check your code against the [coding conventions](/doc/coding-convention.md) and [clang-format](../.clang-format) and [clang-tidy](../.clang-tidy)
- - Clean your code and remove files that are not needed
- - Write documentation related to your new feature if applicable
- - Create a pull request and write a great description about it: what does your PR do, why, how,... Add pictures and video if possible
- - Wait for someone to review your PR and take part in the review process
- - Your PR will eventually be merged :)
+- [Report bugs](https://github.com/InfiniTimeOrg/InfiniTime/issues/new?assignees=&labels=bug&template=bug-report.yaml)
+- Write and improve documentation
+  - Documentation might be incomplete, or not clear enough, and it is always possible to improve it with better wording, pictures, photo, video,...
```
Comment (FintasticMan) [inline doc/contribute.md:5]: Pictures and photos? Maybe best to just pick one.

```suggestion
  - Documentation might be incomplete, or not clear enough, and it is always possible to improve it with better wording, pictures, video,...
```

File: doc/contribute.md [hunk 2]
```diff
@@ -1,61 +1,19 @@
...
+  - As the documentation is part of the source code, you can submit changes to the documentation by creating a pull request (see below).
+- Fix bugs, add functionalities and improve the code
+  - You want to fix a bug, add a cool new functionality or improve the code? See *How to create a pull request below*.
```
Comment (FintasticMan) [inline doc/contribute.md:8]: I don't think that below should also be italicised here.

```suggestion
  - You want to fix a bug, add a cool new functionality or improve the code? See *How to create a pull request* below.
```

**Review by FintasticMan — APPROVED** [review]
Seems like a nice improvement, this should lower the bar for entry to contributing.

File: doc/contribute.md [hunk 3]
```diff
@@ -1,61 +1,19 @@
...
+- [Report bugs](https://github.com/InfiniTimeOrg/InfiniTime/issues/new?assignees=&labels=bug&template=bug-report.yaml)
+- Write and improve documentation
+  - Documentation might be incomplete, or not clear enough, and it is always possible to improve it with better wording, pictures, videos,...
+  - As the documentation is part of the source code, you can submit changes to the documentation by creating a pull request (see below).
+- Fix bugs, add functionalities and improve the code
+  - You want to fix a bug, add a cool new functionality or improve the code? See *How to create a pull request* below.
+
+## How to create a pull request?
+
+- Fork the project, create a [feature branch](branches.md) from develop and give it a short name that explains the topic of your changes.
```
Comment (NeroBurner) [inline doc/contribute.md:12]: not all sentences have a trailing `.`. I think we should add a `.` or remove them for all

for me I'd remove the trailing `.` altogether

**Review by NeroBurner — CHANGES_REQUESTED** [review]

File: doc/contribute.md [hunk 4]
```diff
@@ -1,61 +1,19 @@
...
+- Write and improve documentation
+  - Documentation might be incomplete, or not clear enough, and it is always possible to improve it with better wording, pictures, videos,...
+  - As the documentation is part of the source code, you can submit changes to the documentation by creating a pull request (see below)
+- Fix bugs, add functionalities and improve the code
+  - See *How to create a pull request* below
+
+## How to create a pull request?
+
+1. Fork the project, create a [feature branch](branches.md) from develop and give it a short name that explains the topic of your changes
+   - Any feature branch should focus on one topic only
```
Comment (NeroBurner) [inline doc/contribute.md:13]: use 2 spaces for indentation

```suggestion
  - Any feature branch should focus on one topic only
```

File: doc/contribute.md [hunk 5]
```diff
@@ -1,61 +1,19 @@
...
+2. Make changes in this branch
+   - Write code that satisfies the [coding conventions](/doc/coding-convention.md)
+   - Test your changes on a PineTime or the [InfiniTime simulator](https://github.com/InfiniTimeOrg/InfiniSim)
```
Comment (NeroBurner) [inline doc/contribute.md:16]: use 2 spaces for indentation

```suggestion
  - Write code that satisfies the [coding conventions](/doc/coding-convention.md)
  - Test your changes on a PineTime or the [InfiniTime simulator](https://github.com/InfiniTimeOrg/InfiniSim)
```

**Review by NeroBurner — CHANGES_REQUESTED** [review]

File: doc/contribute.md (hunk 4 above)
Comment (Riksu9000) [reply doc/contribute.md:13]: The first character of the item must align with the first character of the nested list marker. See https://stackoverflow.com/questions/49278202/markdown-enumerated-list-with-indices-over-99-changes-nested-list-formatting/49302511#49302511

Comment (NeroBurner) [reply doc/contribute.md:13]: First time I heard of it, and I don't like it :see_no_evil: But anyways, marking as resolved

**Review by NeroBurner — APPROVED** [review]

**Review by JF002 — APPROVED** [review]

**Pull request merged**